CONTRIBUTING.md

```diff
@@ -43,6 +43,14 @@ Don't forget to add your new article to the `SUMMARY.md` to let it be rendered t
 
 Please make `Draft Pull requests` early so we can follow your progress and can give early feedback (see the following section).
 
+## Style guide
+
+- Follow the book [style guide](https://github.com/rust-lang/book/blob/master/style-guide.md).
+- Follow [RFC 1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text). TL;DR:
+  - Use American English
+  - Use full types name. For example `Option<T>` instead of `Option`
+  - Prefer line comments (`//`) over block comments (`/* */`)
```

Comment:
I think this is pretty much standard that we don't need to mention this already, I wonder if we have even came across this?

Reply:
Yes, it's standard, but it doesn't hurt to put it there if there are new people coming from different languages I think.
Does anybody else have an opinion on this?

Reply:
i saw block comments used in arguments to keep the argument brackets of functions on the same line. they are sometimes more useful than line comments. I'm not sure if we can adhere to this style completely. But in general it might be not bad to state it maybe just telling the people "hey, look at the markdown of another document and copy the style of the code" or so?

Reply:
> i saw block comments used in arguments to keep the argument brackets of functions on the same line. they are sometimes more useful than line comments. I'm not sure if we can adhere to this style completely.

It is just the preferred style, there can be exceptions.

> hey, look at the markdown of another document and copy the style of the code

I think that's what people do by default.

The reason why I put the TL;DR there is that I wanted to summarize what the RFC says in order to avoid forcing people to read all the RFC and only put what's relevant for us.

Do you agree to keep this line there?

Reply:
```suggestion
  - Prefer line comments (`//`) over block comments (`/* */`) where applicable
```

Like this maybe?

Reply:
fixed in https://github.com/rust-unofficial/patterns/pull/201/commits/50c066357eab9601ae297b5adb62444d2fd5742c